ci/prow-entrypoint.sh

```diff
@@ -123,12 +123,21 @@ kola_test_qemu() {
 
     # Skip Secure Boot tests on SCOS for now
     # See: https://github.com/openshift/os/issues/1237
-    if rpm-ostree compose tree --print-only "${manifest}" | jq -r '.packages[]' | grep -q "centos-release"; then
-        cosa kola --basic-qemu-scenarios --output-dir ${ARTIFACT_DIR:-/tmp}/kola-basic
+    # Due to the changes in https://github.com/coreos/coreos-assembler/pull/3652
+    # we need to check if the basic.nvme is available in the list of tests
+    local args=""
+    if cosa kola list --json | jq -r '.[].Name' | grep -q "basic.nvme"; then
+        if ! rpm-ostree compose tree --print-only "${manifest}" | jq -r '.packages[]' | grep -q "centos-release"; then
```

Comment:
While reviewing this, I noticed https://github.com/openshift/os/pull/1448/files#r1526385732. I guess we might as well fix it here. So we can drop the `!` and define `manifest=src/config/manifest.yaml` just above this.

And similarly in the else block below, either add a `!` or swap the if-statement body with the else body.

Reply:
I think we should drop this workaround now that we have proper filtering and use the kola denylist instead.

Reply:
@jlebon  ok, done :)

@travier could you elaborate? Do you mean we should update the kola tests to not run on centos?